build_system/src/config.rs

```diff
@@ -469,11 +463,28 @@ impl ConfigInfo {
 }
 
 fn download_gccjit(
-    url: String,
+    commit: &str,
     output_dir: &Path,
     tempfile_name: String,
     with_progress_bar: bool,
 ) -> Result<(), String> {
+    let url = if std::env::consts::OS == "linux" || std::env::consts::ARCH == "x86_64" {
+        format!("https://github.com/rust-lang/gcc/releases/download/master-{}/libgccjit.so", commit)
```

Comment:
Thanks! :)

Reply:
Oh, looking at this I just noticed that I messed up the condition it should be and not or.

Reply:
Oh indeed! Completely missed it as well. ^^'